mlir/lib/Conversion/SCFToOpenMP/SCFToOpenMP.cpp

```diff
@@ -390,6 +392,12 @@ struct ParallelOpLowering : public OpRewritePattern<scf::ParallelOp> {
 
     // Create the parallel wrapper.
     auto ompParallel = rewriter.create<omp::ParallelOp>(loc);
+    if (numThreads > 1) {
+      rewriter.setInsertionPoint(ompParallel);
+      mlir::Value numThreadsVar = rewriter.create<LLVM::ConstantOp>(
```

Comment:
Nit: no need to prefix with `mlir::` inside mlir codebase.

Reply:
Right, thanks!